### ISSUE TITLE
GraphQL field type for common field properties

### ISSUE THREAD

**Issue by jaydensmith** [verbb/formie#193]
**What are you trying to do?**
When querying a complex form in GraphQL, currently you need to specify a lot of common field across different field types.

```graphql
query ($handle: [String]!) {
    form(handle: $handle) {
        id
        title
        pages {
            id
            fields {
                name
                handle
                required
                displayName
                instructions
                columnWidth

                ... on Field_Section {
                    placeholder
                    limitAmount
                    limitType
                    errorMessage
                    labelPosition
                    instructionsPosition
                    cssClasses
                }

                ... on Field_Email {
                    placeholder
                    limitAmount
                    limitType
                    errorMessage
                    labelPosition
                    instructionsPosition
                    cssClasses
                }

                ... et al.
```

**What's your proposed solution?**
A field type with the common properties that's shared across all applicable fields.

**Comment by engram-design** [comment]
My only real concern is there's a very slim amount of properties that are applicable across all fields. For instance, a Section field doesn't have half of the properties you're using above. I'll see what I can do.

**Comment by jaydensmith** [comment]
@engram-design yep for some that's true, I was thinking for those they could just be null or blank? Just trying to simplify this enormous query and make it a little more portable.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.24",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.2.25](https://github.com/verbb/formie/releases/tag/1.2.25)
